PySDM/backends/impl_numba/methods/collisions_methods.py

```diff
@@ -114,8 +114,8 @@ def break_up(  # pylint: disable=too-many-arguments,unused-argument,too-many-loc
     atomic_add(breakup_rate, cid, gamma_tmp * multiplicity[k])
     atomic_add(breakup_rate_deficit, cid, gamma_deficit * multiplicity[k])
     # perform rounding as necessary
-    multiplicity[j] = round(nj)
-    multiplicity[k] = round(nk)
+    multiplicity[j] = max(round(nj), 1)
```

Comment:
CC @abulenok 
this change make the Numba backend diverge from the GPU one (under construction)
let's add a unit test so that we have a way of ensuring that it is present in both backends

Reply:
should this change also be introduced in `break_up_while` method?

Reply:
Yes it should also be changed there. We need to ensure multiplicity doesn't get set to zero in this rounding step.